android-nrf-mesh-library/meshprovisioner/src/main/java/no/nordicsemi/android/meshprovisioner/MeshProvisioningHandler.java

```diff
@@ -281,9 +295,24 @@ void identify(@NonNull final UUID uuid, final String nodeName, @NonNull final Ne
      */
     void startProvisioning(@NonNull final UnprovisionedMeshNode unprovisionedMeshNode) throws IllegalArgumentException {
         this.attentionTimer = 0x05;
+        this.usedAuthenticationOOBType = AuthenticationOOBMethods.OUTPUT_OOB_AUTHENTICATION;
```

Comment:
Is there any reason why this value is hard coded to use OOB authentication? as some devices may not provide any oob at all.

Reply:
Okay, I see now. I've not tested using without oob actions... I wanted to keep the default behaviour, but one would need to check here if output is even available. Sorry for this quick and dirty solution, we should probably unify this and have not only oob type but also action type  as parameters (in case one would like to use beep instead of blink).